woocommerce-customer-order-coupon-export/csv/re-order-order-export-columns.php

```diff
@@ -13,11 +13,11 @@ function sv_wc_csv_export_reorder_columns( $column_headers ) {
 	// remove order total from the original set of column headers, otherwise it will be duplicated
 	unset( $column_headers['order_total'] );
 
-	$new_column_headers = array();
+	$new_column_headers = [];
 
 	foreach ( $column_headers as $column_key => $column_name ) {
 
-		$new_column_headers[ $column_key ] = $column_name;
+		$new_column_headers[$column_key] = $column_name;
```

Comment:
@stevemoen we do want to keep these spaces as per the [WordPress PHP Coding Standards - Space Usage](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage)